kalite/distributed/static/js/distributed/rating/views.js

```diff
@@ -203,19 +210,8 @@ var TextView = BaseView.extend({
         return this;
     },
 
-    text_changed: function() {
-        this.model.set(this.text_attr, this.$(".rating-text-feedback")[0].value);
-    },
-
-    toggle_disabled: function(val) {
-        if( typeof val === "undefined" ) {
-            val = !this.is_disabled();
-        }
-        this.$(".rating-text-feedback").attr("disabled", val);
-        return this;
-    },
-
-    is_disabled: function() {
-        return this.$(".rating-text-feedback").attr("disabled") === "disabled";
-    }
+    text_changed: _.throttle(function() {

```

Comment:
@jamalex 
Ok, I'll change it, but just to reiterate [what I said above](https://github.com/learningequality/ka-lite/pull/4248#discussion_r37657574): if someone navigates away from a content item in the 2s window after they stop typing, then when the debounced `save` is called it will not sync whatever text they have entered -- instead of creating a new view every time the user navigates to a different item, only the model object is switched out, so the debounced view method uses the new model attributes. It might be possible to instantiate a new view and keep a reference to the old one around until it's fired off any debounced functions queued up, or even to make this a model method and keep old models around in a similar way. Simply decreasing the debounce delay still leaves the potential for a race condition, but decreases the chance that a human will notice it. What do you think?

Just to make sure I tried it out, and it's not particularly challenging to cause the sync to fail as I've described with a 2s delay. Unlikely for a person to do IMO, but something which could be quite unexpected in automated contexts.